pom.xml

```diff
@@ -844,6 +847,11 @@
         <plugins>
           <plugin>
             <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+            	<parallel>classes</parallel>
```

Comment:
Please fix formatting to match the line above - use 2 instead 1 spaces. Same for the line below